usr.bin/netstat/if.c

```diff
@@ -705,7 +705,7 @@ sidewaysintpr(void)
 	    new->ift_ip - old->ift_ip, 1, 1);
 	show_stat("lu", 5, "received-errors",
 	    new->ift_ie - old->ift_ie, 1, 1);
-	show_stat("lu", 5, "dropped-packets",
+	show_stat("lu", 5, "dropped-packets-in",
```

Comment:
Convert `dropped-packets` to `dropped-packets-in` breaks IMO. I'd prefer keep `dropped-packets` as is, and add redundant `dropped-packets-in`. The `dropped-packets` can be eventually removed in future release.

Reply:
Done. In the future I will add a `xo_set_version` to this command so that users can track changes in the content.